src/microbe_stage/editor/CellEditorComponent.tscn

```diff
@@ -1024,6 +1028,10 @@ layout_mode = 2
 [node name="CompoundStorageStatistics" parent="RightPanel/VBoxContainer/Statistics/VBoxContainer/Body/ScrollContainer/MarginContainer/VBoxContainer" instance=ExtResource("28_vnd4w")]
 layout_mode = 2
 
+[node name="ProcessListButton" type="Button" parent="RightPanel/VBoxContainer/Statistics/VBoxContainer/Body/ScrollContainer/MarginContainer/VBoxContainer"]
+layout_mode = 2
+text = "PROCESS_PANEL_TITLE"
```

Comment:
A button shouldn't have a text key on it that ends in "TITLE" as that makes no sense and can make translating this impossible. I recommend changing the translation key to "VIEW_CELL_PROCESSES" or something like that.

Reply:
I wanted to use already existing translation.

Reply:
It's possible in some cases, but not in this as a button is an action so a translation key that is a static title should not be reused here.